packages/flutter_tools/gradle/app_plugin_loader.gradle

```diff
@@ -23,6 +23,10 @@ assert object.plugins.android instanceof List
 object.plugins.android.each { androidPlugin ->
   assert androidPlugin.name instanceof String
   assert androidPlugin.path instanceof String
+  def needsBuild = androidPlugin.containsKey('native_build') ? androidPlugin['native_build'] : true
```

Comment:
nit: could it have a comment about why it's ok to skip these android plugins?

Reply:
Done.